app/controllers/mfa_confirmation_controller.rb

```diff
@@ -19,6 +19,9 @@ def new
   end
 
   def create
+    irs_attempts_api_tracker.logged_in_password_change_reauthentication_submitted(
+      success: current_user.valid_password?(password),
+    )
     if current_user.valid_password?(password)
```

Comment:
how expensive is `valid_password?` calculation? I think we use an intentionally slow password hashing mechanism... so I would recommend we save the result and don't call twice

```suggestion
    valid_password = current_user.valid_password?(password)
    irs_attempts_api_tracker.logged_in_password_change_reauthentication_submitted(
      success: valid_password,
    )
    if valid_password
```

Reply:
I nearly made that change before pushing up this PR.  Good call!